packages/charts/src/charts/bubble/bubble-chart.component.tsx

```diff
@@ -48,8 +48,8 @@ export const BubbleChart: FC<Props> = ({
   valueKey,
   theme = defaultTheme,
   margins = defaultMargins,
-  maxAreaRadius = 20,
```

Comment:
In types definition that both values are not required.
```
 minAreaRadius?: number;
 maxAreaRadius?: number;
``` 
Removing fallback to default values will cause an error for use case when only `BubbleChart` plot is used.